src/components/menu/menu/menu.component.ts

```diff
@@ -94,6 +99,11 @@ export class MenuComponent implements AfterContentInit, OnDestroy, OnChanges {
     /** Automatically unsubscribe when the component is destroyed */
     private readonly _onDestroy$ = new Subject<void>();
 
+    /** Get the aria controls label */
+    get _ariaControls(): string {
```

Comment:
Since this is used outside of the component, it doesn't need the underscore - we use that for properties which are public only for the purposes of the component's template. Also I would suggest something like `innerId` because it is only an `aria-controls` value from the perspective of the other control.